docs/Systems/Tailwind/UtilityClassReferences/Borders/border-color.stories.tsx

```diff
@@ -1,10 +1,9 @@
 import React from "react"
-import { Story } from "@storybook/react"
+import { Meta, StoryFn } from "@storybook/react"
 import classnames from "classnames"
 import { kaizenTailwindTheme } from "@kaizen/tailwind"
-import { CATEGORIES } from "../../../../../storybook/constants"
 import { UtilityClassTemplate } from "../../components/UtilityClassTemplate"
-import { flattenEntries } from "../../helpers/flatten-entries"
+import { flattenEntries } from "../../helpers/flattenEntries"
```

Comment:
Was this named incorrectly the whole time? I don't see a file rename lol. I think kebab-case is nicer, but not important right now.

Reply:
There's another util in the same folder which is camel case 😂 For utils and components, I like to match the function name/component name. I'd only use kebab-case if the contents do not match the file name (eg. it's a category of many things).